docs/_config.yml

```diff
@@ -8,7 +8,7 @@ header_pages:
 
 markdown: kramdown
 
-repository: "se-edu/addressbook-level3"
+repository: "AY2223S1-CS2103-F09-2/tp"
```

Comment:
Hmm could I ask what does line does as the GitHub icon on the deployment website still links back to the tp repo.

Reply:
This line is supposed to change the URL of the GitHub icon on the header bar to point to our team's repo.

The website that is deployed is taken from the files in the `master` branch, not from this PR. So the URL should update once we merge this PR with `master`.